code/model/editableformfields/EditableTextField.php

```diff
@@ -46,6 +47,45 @@ public function getCMSFields()
                     _t('EditableTextField.PLACEHOLDER', 'Placeholder')
                 )
             );
+
+            $fields->addFieldToTab(
+                'Root.Main',
+                DropdownField::create(
+                    'Autocomplete',
+                    'HTML5 Autocomplete value',
+                    array(
+                      'off' => 'Off',
+                      'on' => 'On',
```

Comment:
form only accepts on/off 
however input allows the attribute to define what autocomplete value should be pre populated as your label may be different to the input name. 

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#attr-autocomplete
https://html.spec.whatwg.org/multipage/forms.html#autofill

both links break valid elements down i haven't included all of them as things like credit card number probably shouldn't be auto completed in a user defined form, but thats purely a judgment call they could easily be in the list.